x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/hosts/risk_score/query.hosts_risk.dsl.ts

```diff
@@ -5,26 +5,30 @@
  * 2.0.
  */
 
-import { HostsRiskyHostsRequestOptions } from '../../../../../../common/search_strategy/security_solution/hosts/risky_hosts';
-import { createQueryFilterClauses } from '../../../../../utils/build_query';
+import { HostsRiskScoreRequestOptions } from '../../../../../../common';
 
-export const buildRiskyHostsQuery = ({
-  filterQuery,
-  timerange: { from, to },
+export const buildHostsRiskScoreQuery = ({
```

Comment:
i dont think this should be plural if its for a single host

Reply:
Addressed over here https://github.com/elastic/kibana/pull/113274#discussion_r722377060